config/locales/en.yml

```diff
@@ -140,7 +140,7 @@ en:
       blank: "Please select a provider"
     national_insurance_number:
       blank: Enter the national insurance number (NINo)
-      invalid: National Insurance number is not the correct format (for example, AB 12 34 56 A or AB123456A)
+      invalid: National Insurance number is not the correct format (for example, QQ 12 34 56 A or QQ123456A)
```

Comment:
I think in this case the example format should be in the hint rather than the error message. There's a similar example in [the design system](https://design-system.service.gov.uk/components/error-message/#label).

Reply:
@peteryates  the hint already have it. See [nino.html](https://github.com/DFE-Digital/early-careers-framework/pull/2393/files#diff-3ee2729e3caa6215a96ae52f457f8cb06749138a5c3a3aca1281994c559ac9e6R20) template. 
I am just sticking to the same pattern as the existing [TRN](https://github.com/DFE-Digital/early-careers-framework/pull/2393/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7L145).
But happy to remove the `(for example, QQ 12 34 56 A or QQ123456A)` if you think it is better that way

Reply:
@peteryates Invalid nino error message updated after talking with @JoDimbleby. Thanks!!